lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +25,21 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
+                                       description: "Carthage command (one of `build`, `bootstrap`, `update`)",
+                                       default_value: 'bootstrap',
+                                       verify_block: proc do |value|
+                                         raise "Please pass a valid command. Use one of the following: build, bootstrap, update" unless %w(build bootstrap update).include? value
+                                       end),
           FastlaneCore::ConfigItem.new(key: :update,
```

Comment:
Maybe we can mark this one as deprecated? Or considering it was added so recently (like, less than a week ago), I think it would be worth to just remove it.